feedback/styles/styles.scss

```diff
@@ -0,0 +1,683 @@
+    color: $white;
+    text-transform: uppercase;
+    letter-spacing: $capsspacing;
+    font-size: $capssize;
+    font-weight: $bold-weight;
+    text-align: center;
+    align-items: center;
+
+    &_link1 {
+        grid-column: 1/2;
+        grid-row: 1/2;
+        padding: 20px;
+        background: $pink;
+    }
+
+    &_link2 {
+        grid-column: 2/3;
+        grid-row: 1/2;
+        padding: 20px;
+        background: $pink;
+    }
+
+    &_link3 {
+        grid-column: 3/4;
+        grid-row: 1/2;
+        padding: 20px;
+        background: $pink;
+    }
+}
+
+
+footer {
+    grid-column: 1/11;
+    grid-row: 19/22;
+    display: grid;
+    grid-template-columns: repeat(4, 1fr);
+    grid-template-rows: repeat(2, 1fr);
+    grid-gap: 3vw;
+    margin: -8px;
+    padding: 4vw;
+    background: $blue;
+    color: $white;
+
+
+    &_sec1 {
+        grid-column: 1/2;
+        grid-row: 1/2;
+        font-size: $capssize;
+        letter-spacing: $capsspacing;
+    }
+
+    &_sec2 {
+        grid-column: 2/3;
+        grid-row: 1/2;
+        font-size: $footersize;
+        padding-top: $footersec_toppadding;
+    }
+
+    &_sec3 {
+        grid-column: 1/2;
+        grid-row: 2/3;
+    }
+
+    &_sec4 {
+        grid-column: 2/3;
+        grid-row: 2/3;
+        font-size: $footersize;
+        padding-top: $footersec_toppadding;
+    }
+
+    &_sec5 {
+        grid-column: 3/4;
+        grid-row: 1/2;
+        padding-left: 3vw;
+    }
+
+    &_sec6 {
+        grid-column: 4/5;
+        grid-row: 1/2;
+        font-size: $footersize;
+        padding-top: $footersec_toppadding;
+    }
+
+    &_sec7 {
+        grid-column: 3/4;
+        grid-row: 2/3;
+    }
+
+    &_sec8 {
+        grid-column: 4/5;
+        grid-row: 2/3;
+        font-size: $footersize;
+        padding-top: $footersec-toppadding;
+    }
+}
+
+h1 {
+    grid-column: 1/6;
+    grid-row: 3/4;
+    padding-left: $left-margin;
+    font-weight: $bold-weight;
+    font-size: 2.5vw;
+    letter-spacing: .1vw;
+    margin-top: 4vw;
+    margin-bottom: 0;
+    color: $white;
+}
+
+h2 {
+    font-size: $capssize;
+    text-transform: uppercase;
+    letter-spacing: $capsspacing;
+}
+
+header {
+    grid-column: 1/10;
+    grid-row: 1/7;
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(10, .5fr);
+    margin-top: -1.1vw;
+    margin-left: -1.1vw;
+    background: $blue;
+}
+
+.heroimage {
+    width: 100%;
+    grid-column: 4/9;
+    grid-row: 4/9;
+    padding-top: 4vw;
+}
+
+
+.menu-icon {
+    &__line {
+        background-color: $white;
+        height: 2px;
+        margin: 8px auto;
+        transition: all 0.3s ease-in-out;
+        width: 42px;
+    }
+}
+
+.mission-statement {
+    grid-column: 1/4;
+    grid-row: 4/6;
+    padding-left: $left-margin;
+    padding-right: 30%;
+    font-size: 2vw;
+    line-height: 3.5vw;
+    font-weight: $regular-weight;
+    color: $white;
+}
+
+.nav {
+    &_item1 {
```

Comment:
You may not need all these styles? Unless you're moving things around a lot, I would verify if this isn't how the links lay out automatically on the grid.